### PR TITLE
Consolidate Inner/Outer Express into single bus fare

### PR DESCRIPTION
**Pull request by hannahpurcell**
#### Summary of changes
**Asana Ticket:** [Fares/Tariff | All inner/outer express bus routes are called "express bus"](https://app.asana.com/0/555089885850811/1200354440910073)

The Inner / Outer Express Bus is consolidated into a single bus.  The fares are rounding down to the "Inner" fare.  

For the most part, this just meant replacing every instance of `:inner_express_bus` with `:express_bus` and removing `:outer_express_bus` entirely.  (The only exception was the transfer logic, in which we kept the Outer Express Bus logic, since a transfer between Express Buses does not involve a transfer fee.)

** One thing not yet tested that will need to be tested:  are the transfer calculations correct?

Things to test:
- Fares are corrected in the [Trip Planner](http://localhost:4001/trip-planner?_utf8=%E2%9C%93&plan%5Bfrom%5D=Woburn%2C+MA%2C+USA&plan%5Bfrom_latitude%5D=42.4792618&plan%5Bfrom_longitude%5D=-71.1522765&plan%5Bto%5D=State+Street+Provisions%2C+State+Street%2C+Boston%2C+MA%2C+USA&plan%5Bto_latitude%5D=42.3595128&plan%5Bto_longitude%5D=-71.05141689999999&plan%5Btime%5D=depart&plan%5Bdate_time%5D%5Bhour%5D=3&plan%5Bdate_time%5D%5Bminute%5D=15&plan%5Bdate_time%5D%5Bam_pm%5D=PM&plan%5Bdate_time%5D%5Bmonth%5D=6&plan%5Bdate_time%5D%5Bday%5D=29&plan%5Bdate_time%5D%5Byear%5D=2021&plan%5Bmodes%5D%5Bsubway%5D=false&plan%5Bmodes%5D%5Bsubway%5D=true&plan%5Bmodes%5D%5Bcommuter_rail%5D=false&plan%5Bmodes%5D%5Bcommuter_rail%5D=true&plan%5Bmodes%5D%5Bbus%5D=false&plan%5Bmodes%5D%5Bbus%5D=true&plan%5Bmodes%5D%5Bferry%5D=false&plan%5Bmodes%5D%5Bferry%5D=true&plan%5Boptimize_for%5D=best_route#plan_result_focus)
- Fares associated with buses previously classified as Outer Express:  http://localhost:4001/schedules/354/line

### CODE REVIEW

File: apps/site/lib/site/content_rewriters/liquid_objects/fare.ex [hunk 1]
```diff
@@ -161,6 +160,10 @@ defmodule Site.ContentRewriters.LiquidObjects.Fare do
     {filter_insert(good, mode: value), bad}
   end
 
+  defp parse_token(value, good, bad) when value in ["inner_express_bus", "outer_express_bus", "express_bus"] do
```
Comment (hannahpurcell) [inline apps/site/lib/site/content_rewriters/liquid_objects/fare.ex:163]:
```suggestion
  defp parse_token(value, good, bad) when value in ["inner_express_bus", "outer_express_bus"] do
```

**Comment by thecristen** [comment]
BTW - Elixir tests should run on this PR on the next run - this PR made me realize I had to commit a fix because it was skipping testing when it wasn't supposed to. 😓

**Comment by amaisan0** [comment]
> BTW - Elixir tests should run on this PR on the next run - this PR made me realize I had to commit a fix because it was skipping testing when it wasn't supposed to. 😓 

Darn- we almost snuck through!

**Review by amaisano — COMMENTED** [review]
LGTM - I'd like to do some quick CMS testing before doing a final approval.

**Review by amaisano — APPROVED** [review]
Checked out the CMS token conversion. Looks good to me! Might want to have a content admin review as well. Thank you @hannahpurcell !

**Comment by hannahpurcell** [comment]
> BTW - Elixir tests should run on this PR on the next run - this PR made me realize I had to commit a fix because it was skipping testing when it wasn't supposed to. 😓

I think "creating a PR" is the only prompt for the Elixir tests.  What's a good way to prompt them to run at this point?

**Comment by thecristen** [comment]
@hannahpurcell boooo! I wonder if closing and re-opening the PR might work? 
if that doesn't work I'll just review the old-fashioned way and checkout the branch locally and run the tests myself. 🤪

File: apps/fares/lib/fare_info.ex [hunk 2]
```diff
@@ -181,18 +181,17 @@ defmodule Fares.FareInfo do
...
+        "express_bus",
         charlie_card_price,
         day_reduced_price,
         _month_reduced_price,
         _day_pass_price,
         _week_pass_price,
         month_pass_price | _
-      ])
-      when mode in ["inner_express_bus", "outer_express_bus"] do
+      ]) do
```
Comment (thecristen) [inline apps/fares/lib/fare_info.ex:191]: It looks like this function's now nearly identical to 
```elixir
def mapper([
        "local_bus",
        charlie_card_price,
        day_reduced_price,
        _month_reduced_price,
        _day_pass_price,
        _week_pass_price,
        month_pass_price | _
      ])
``` 
now? So I guess we can consolidate them now (much like how `mode in ["inner_express_bus", "outer_express_bus"]` handled this before 🙈 ).

**Review by thecristen — APPROVED** [review]
I ran the unit tests locally (there are a smattering of failing Jest and Elixir unit tests but they are unrelated to these changes) and so it LGTM!

File: apps/fares/lib/fare_info.ex (hunk 2 above)
Comment (hannahpurcell) [reply apps/fares/lib/fare_info.ex:191]: Oh yeah!  I guess they all could have been consolidated all along.  I like it!

**Pull request merged**